.github/workflows/recover_s3_repository_periodic_test.yml

```diff
@@ -0,0 +1,135 @@
+name: . ⚠️🔧 Test Recover S3 Repository back in time 🔧️⚠️
+
+on:
+  workflow_dispatch:
+  schedule:
+   #Scheduled to run at 7 a.m on every day-of-week from Monday through Friday.
+    - cron: "0 7 * * 1-5"
```

Comment:
I don't think it does, but I'd like to double-check: could the testing-recovering collide somehow with any on-going automatic (pre-)release?

Reply:
Nope. This uses a custom directory not colliding with anything else.

https://github.com/newrelic/infrastructure-agent/pull/2015/files#diff-e20390ab72cb991c3764580f22dc7f190f38edc407d85dd9dd9b606a67f7d853R10